_spinnaker_user_guides/plugin-users.md

```diff
@@ -28,48 +34,64 @@ resources:
-`hal plugins enable`
+## Enabling Plugins for Spinnaker
+
+To enable plugins for your Spinnaker deployment run the following [command](https://www.spinnaker.io/reference/halyard/commands/#hal-plugins-enable):
+
+```
+hal plugins enable
+```
+
+This enables plugins to be loaded if the individual plugin(s) are enabled.
 
-This will enable plugins to be loaded if the individual plugin(s) are enabled.
+## Adding a Plugin
 
-## Adding A Plugin
+To add a specific plugin to your deployment, use the `add` command and provide the plugin name and manifest location:
 
-To add a plugin to Spinnaker, the plugin manifest location needs to be known. Adding the plugin is as easy as:
 ```
-hal plugins add plugin-name --enabled \
+hal plugins add <plugin-name> --enabled \
      --manifest-location="https://path/to/plugin/manifest.yml"`
 ```
 
-The `--enabled` is to automatically enable the plugin. Plugins by default are disabled, unless the `--enabled` flag is passed when adding the plugin.
+The `--enabled` flag automatically enables the plugin. Plugins, by default, are disabled, unless the `--enabled` flag is passed when adding the plugin.
 
-**Applying Changes**
-Anytime things via Halyard change, the command `hal deploy apply` needs to be ran to send the configuration off to Spinnaker. Plugins are no different. After modifying any plugin remember to run `hal deploy apply` for the changes to be passed to Spinnaker.
+## Controlling Plugin Downloads
 
-## Controlling Plugin Downloading
+If Spinnaker is deployed to Kubernetes, you must configure Spinnaker to download the plugin resources. To enable plugin downloads, run the following command:
 
-If Spinnaker is deployed to Kubernetes, Halyard has to enable Spinnaker to download the plugin resources. To do that, run the following command to enable plugin downloading:
-`hal plugins enable-downloading` 
+```
+hal plugins enable-downloading 
+```
 
-If Spinnaker is deployed to something else besides Kubernetes, plugin resources will have to be manually added to the correct locations.
+If Spinnaker is deployed to something besides Kubernetes, plugin resources need to be added to the correct locations.
 
 ## Modifying Plugins
 
-To modify an existing plugin, the `hal plugins edit` [command](https://www.spinnaker.io/reference/halyard/commands/#hal-plugins-edit) is where we recommend to start. Add the plugin name that needs to be modified to the command and then the manifest location can be modified.
+To modify an existing plugin, use the `hal plugins edit` [command](https://www.spinnaker.io/reference/halyard/commands/#hal-plugins-edit) command. Add the plugin name that needs to be modified to the command.
 
-For example, to disable a plugin, run the following command
-`hal plugins edit plugin-name` `--``disable` 
+For example, to disable a plugin, run the following command:
 
-Remember to run `hal deploy apply` after running any modifications
+```
+hal plugins edit <plugin-name> --disable 
```

Comment:
`--disable` is not one of the parameters listed here https://www.spinnaker.io/reference/halyard/commands/#hal-plugins-edit

There's only `--enabled`. Is it `--enabled false'? Or do we need to update `plugins edit` page? 

Also, if it is `--disable` can we remove the 'd' on --enabled so it's consistent or add a 'd; to disable so that they're consistent?

Reply:
Ok thanks. We should clarify that the value is a boolean in the hal command docs then. Can you submit that PR, and I'll approve and merge. 

Also, it's --enable**d**="false", right? since it's --enable**d**="true".

Reply:
It is. Realized I could just poke it myself in Halyard lol.